File: app/src/main/java/com/challenge/fidoreader/fagment/AuthenticatorFragment.java
package com.challenge.fidoreader.fagment;

import android.os.Bundle;

import androidx.annotation.NonNull;
import androidx.annotation.Nullable;
import androidx.fragment.app.Fragment;

import android.view.LayoutInflater;
import android.view.View;
import android.view.ViewGroup;
import android.widget.Button;

import com.challenge.fidoreader.R;


public class AuthenticatorFragment extends Fragment {
    @Nullable
    @Override
    public View onCreateView(@NonNull LayoutInflater inflater, @Nullable ViewGroup container, @Nullable Bundle savedInstanceState) {
        View view = inflater.inflate(R.layout.fragment_authenticator, container, false);

        Button btn = (Button)view.findViewById(R.id.readerActivationBtn);
        btn.setOnClickListener(new View.OnClickListener() {
            @Override
            public void onClick(View view) {
                //TODO
            }
        });

        return view;
    }
}

File: app/src/main/java/com/challenge/fidoreader/frag/reader/AuthenticatorFragment.kt
package com.challenge.fidoreader.frag.reader

import android.os.Bundle
import android.view.LayoutInflater
import android.view.View
import android.view.ViewGroup
import android.widget.Button
import androidx.fragment.app.Fragment
import com.challenge.fidoreader.R

class AuthenticatorFragment : Fragment() {
    override fun onCreateView(inflater: LayoutInflater, container: ViewGroup?, savedInstanceState: Bundle?): View? {
        val mainview = inflater.inflate(R.layout.fragment_authenticator, container, false)
        val btn = mainview.findViewById<View>(R.id.readerActivationBtn) as Button
        btn.setOnClickListener {
            //TODO
        }
        return view
    }
}